client/lib/catch-js-errors/index.js

```diff
@@ -120,12 +119,13 @@ export default class ErrorLogger {
 	}
 
 	sendToApi( error ) {
-		const xhr = new XMLHttpRequest();
+		const params = new URLSearchParams( {
+			error: JSON.stringify( error ),
+		} );
+
+		const xhr = new window.XMLHttpRequest();
 		xhr.open( 'POST', 'https://public-api.wordpress.com/rest/v1.1/js-error?http_envelope=1', true );
 		xhr.setRequestHeader( 'Content-type', 'application/x-www-form-urlencoded' );
-		let params =
-			'client_id=39911&client_secret=cOaYKdrkgXz8xY7aysv4fU6wL6sK5J8a6ojReEIAPwggsznj4Cb6mW0nffTxtYT8&error=';
-		params += encodeURIComponent( JSON.stringify( error ) );
-		xhr.send( params );
+		xhr.send( params.toString() );
```

Comment:
Nice to have: refactor this to use `fetch`.

Reply:
Good call! Maybe the next time we modify this code is in 4.5 years, so it's good to wrangle this while we're here. 😉 

Done in 1e6c9de274739cae123ee38a2c1a9d448b4adbf9 - feel free to take another look before we 🚢 .